src/utils/test/file_utils.cpp

```diff
@@ -59,9 +59,9 @@ static void file_utils_test_get_process_image_path()
         EXPECT_TRUE(false);
     }
 #ifdef WIN32
```

Comment:
We don't plan to keep compatiable with Windows platform, you can remove these ralated code.